packages/core/src/sass/account-wizard.scss

```diff
@@ -59,6 +60,12 @@
             font-weight: bold;
             text-decoration: none;
         }
+        span.chat-inline {
```

Comment:
Do we need the same css in two places? here and inside `account-wizard--error`

Reply:
@oskar-binary  actually `span.chart-inline` is within `account-wizard--error` but the problem is that it only handled the desktop, on mobile we need to `display: block` the `.chart-inline` so that it stands out in a separate line. here is for desktop, the bottom CSS code, is for mobile.

Reply:
@oskar-binary should I handle the mobile CSS in another way?

Reply:
as discussed, it's ok.